server/routes/gitlab/gitlab_routes.py

```
@@ -0,0 +1,287 @@
+        if resp.status_code == 200:
+            return resp.json()
+
+        resp = requests.get(
+            f"{base_url}/api/v4/user",
+            headers=headers,
+            timeout=GITLAB_TIMEOUT,
+        )
+        if resp.status_code == 200:
+            return resp.json()
+    except requests.RequestException as e:
+        logger.error(f"GitLab token validation failed: {e}")
```

Ruff (0.15.12)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)